include/fastrtps/types/TypeObjectHashId.h

```diff
@@ -187,6 +187,10 @@ class TypeObjectHashId
      */
     void serializeKey(eprosima::fastcdr::Cdr &cdr) const;
 
+    uint8_t get_m_d()const noexcept{ return m__d ;}
+    void set_m_d(uint8_t m_d) noexcept{ m__d = m_d;}
+    octet* get_m_hash() noexcept{ return &m_hash[0]; }
+
```

Comment:
Similar (and standard) methods are already defined for this class:

    uint8_t _d() const;
    void _d(uint8_t __d);
    const EquivalenceHash& hash() const;

Why did you create these versions?